pkg/instrumentation/sdk.go

```diff
@@ -56,39 +56,48 @@ type sdkInjector struct {
 	client client.Client
 }
 
-func (i *sdkInjector) inject(ctx context.Context, insts languageInstrumentations, ns corev1.Namespace, pod corev1.Pod) corev1.Pod {
+func (i *sdkInjector) inject(ctx context.Context, insts languageInstrumentations, ns corev1.Namespace, pod corev1.Pod, containerName string, init bool) corev1.Pod {
```

Comment:
Why do we need to pass the last `init` parameter? I don't think it is needed.

Reply:
I do it to keep code as similar as possible with existing one. 
As i call multiple time the injectXXXAgent functions and as this functions also contains the injection of Volume that host agent and init container that copy this agent, i use this simple boolean to avoid this injection multiple time.
The creation of volume and initContainers is only do one time for the first eligible container of injection.

Reply:
The function takes pod that can be used to verify what needs to be ingected

Reply:
You are right i don't think to use it. 
I will check existence of init containers and volumes by adding anoter function i will call from injectXXXAgent.